### ISSUE TITLE
Need help in downloading embedded PDF in viewer 

### ISSUE THREAD

**Issue by Yvttsplvd** [drathier/stack-overflow-import#25]
Not sure where to post this but I have a subscription for magazines but they are in read online only. No other options. Can someone help isolating the url? Thank you.

**Comment by WieeRd** [comment]
@Yvttsplvd I'm genuinely confused, what are you even talking about? This is beyond off-topic.

**Comment by WieeRd** [comment]
Are you by any chance confusing this repository with the actual StackOverflow?

**Comment by Yvttsplvd** [comment]
Yes haha I’m beyond confused on where to actually post this. I just need to know how I can extract a pdf from an embedded pdf viewer. Can you direct to the correct place? 
Thank you. 


On Tuesday, March 5, 2024, 3:40 AM, WieeRd ***@***.***> wrote:




Are you by any chance confusing this repository with the actual StackOverflow?

**Comment by WieeRd** [comment]
@Yvttsplvd <https://stackoverflow.com/>
This is the StackOverflow you were looking for, the programmers' Q&A forum.
But you'll need to your question more detailed and concrete than "Extract a PDF from an embedded PDF viewer".

**Comment by Yvttsplvd** [comment]
Thank you for that link. 
Short of posting the screenshot, I don’t know what details to include. What would be needed? 


On Wednesday, March 6, 2024, 10:07 AM, WieeRd ***@***.***> wrote:




@Yvttsplvd https://stackoverflow.com/
This is the StackOverflow you were looking for, the programmers' Q&A forum.
But you'll need to your question more detailed and concrete than "Extract a PDF from an embedded PDF viewer".
